File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int list[] = new int[n];
        for (int i = 0; i < n; i++) {
            list[i] = sc.nextInt();
        }
        int max = list[0];
        int min = list[0];
        for (int i = 1; i < n; i++) {
            if(list[i] > max) max = list[i];
            if(list[i] < min) min = list[i];
        }
        System.out.print(min + " ");
        System.out.print(max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include<bits/stdc++.h>
using namespace std;
int _max = -1000000, _min = 1000000, n, a;
int main(){
    ios_base::sync_with_stdio(false); cin.tie(NULL); cout.tie(NULL);
    cin >> n;
    for(int i = 0; i < n; i++){
        cin >> a;
        _max = max(_max, a);
        _min = min(_min, a);
    }
    cout << _min << " " << _max;
}